qa/tasks/cephfs/test_exports.py

```diff
@@ -132,8 +132,8 @@ def test_export_pin_getfattr(self):
             p = self.mount_a.client_remote.run(args=['uname', '-r'], stdout=StringIO(), wait=True)
```

Comment:
can this line be changed to:
```
client_uname = self.mount_a.client_remote.sh('unman -r', wait=True)
```
and later:
```
if client_uname < "5" and not(dir_pin):
. . .
```

Reply:
I would like to avoid that in this PR. There are plenty of `StringIO` imports left that are not py3 compatible. Can we tackle these in a different PR?

Reply:
No longer part of this PR so marking this as resolved